### PR TITLE
Modified STL function to include automated seasonal component

### PR DESCRIPTION
**Pull request by aseemanand**
- Modified STL function to include automated seasonal component
- Version update to 0.0.34

### CODE REVIEW

File: src/pycatcher/catch.py [hunk 1]
```diff
@@ -513,9 +513,9 @@ def generate_outliers_stl(df, type, period) -> pd.DataFrame:
 
     if type == 'additive':
         logging.info("Outlier detection using STL Additive Model")
-        stl = STL(df, period=period)
+        df_add = df.copy()
```
Comment (jpamarthi) [inline src/pycatcher/catch.py:516]: Just curious to know why do we need a new clone?

**Review by jpamarthi — APPROVED** [review]
LGTM

**Comment by jpamarthi** [comment]
I will take care of these gaps in the next PR
image

File: src/pycatcher/catch.py (hunk 1 above)
Comment (aseemanand) [reply src/pycatcher/catch.py:516]: For better readability/reference about different logic for additive and multiplicative seasonalities :)

Comment (jpamarthi) [reply src/pycatcher/catch.py:516]: Ah! Ok. I need to check on the memory usage when doing some performance testing.

**Pull request merged**